source/renderer/app/components/wallet/send-form/WalletSendConfirmationDialog.js

```diff
@@ -379,10 +314,9 @@ export default class WalletSendConfirmationDialog extends Component<
               />
             </div>
           )}
-
           {this.renderConfirmationElement(isHardwareWallet)}
+          {shouldShowEmptyWalletWarning(totalAmount, wallet) && walletWarning}
```

Comment:
@danielmain I like this approach much better than the one you did in the `source/renderer/app/components/wallet/send-form/WalletSendAssetsConfirmationDialog.js`
Can we unify these?